data/card_definitions.json

```diff
@@ -10457,6 +10457,324 @@
 	},
 
 
+	{
+		"NOTE": "MOLE CARDS START",
+		"id": "mole_erupt",
+		"type": "ultra",
+		"display_name": "Erupt",
+		"force_cost": 0,
+		"gauge_cost": 3,
+		"range_min": 2,
+		"range_max": 4,
+		"speed": 1,
+		"power": 8,
+		"armor": 2,
+		"guard": 5,
+		"effects": [
+		],
+		"boost": {
+			"boost_type": "continuous",
+			"force_cost": 1,
+			"cancel_cost": -1,
+			"display_name": "Molten Armor",
+			"effects": [
+				{
+					"timing": "during_strike",
+					"effect_type": "rangeup",
+					"amount": 0,
+					"amount2": 1
+				},
+				{
+					"timing": "when_hit",
+					"first_time_only": true,
+					"effect_type": "take_damage",
+					"opponent": true,
+					"nonlethal": true,
+					"amount": 2
+				}
+			]
+		}
+	},
+	{
+		"id": "mole_cavein",
+		"type": "ultra",
+		"display_name": "Cave In",
+		"force_cost": 0,
+		"gauge_cost": 2,
+		"range_min": 1,
+		"range_max": 3,
+		"speed": 1,
+		"power": 4,
+		"armor": 3,
+		"guard": 0,
+		"effects": [
+			{
+				"timing": "during_strike",
+				"effect_type": "opponent_cant_move_if_in_range"
+			},
+			{
+				"timing": "after",
+				"effect_type": "draw",
+				"amount": 2
+			}
+		],
+		"boost": {
+			"boost_type": "continuous",
+			"force_cost": 0,
+			"cancel_cost": -1,
+			"display_name": "Undermine",
+			"effects": [
+				{
+					"timing": "start_of_next_turn",
+					"effect_type": "choice",
+					"choice": [
+						{
+							"effect_type": "discard_this",
+							"bonus_effect": {
+								"condition": "opponent_on_buddy_space",
+								"condition_detail": "Burrow",
+								"effect_type": "take_damage",
+								"opponent": true,
+								"nonlethal": true,
+								"amount": 4
+							}
+						},
+						{
+							"effect_type": "move_buddy",
+							"buddy_name": "Burrow",
+							"amount": 1,
+							"amount2": 1
+						}
+					]
+				}
+			]
+		}
+	},
+	{
+		"id": "mole_burrowdig",
+		"type": "special",
+		"display_name": "Burrow Dig",
+		"force_cost": 0,
+		"gauge_cost": 0,
+		"range_min": 1,
+		"range_max": 1,
+		"speed": 7,
+		"power": 3,
+		"armor": 0,
+		"guard": 0,
+		"effects": [
+			{
+				"timing": "hit",
+				"effect_type": "draw",
+				"amount": 1,
+				"and": {
+					"effect_type": "advance",
+					"amount": 2
+				}
+			}
+		],
+		"boost": {
+			"boost_type": "immediate",
+			"force_cost": 0,
+			"cancel_cost": -1,
+			"display_name": "Lost Treasure",
+			"effects": [
+				{
+					"timing": "immediate",
+					"effect_type": "choose_discard",
+					"limitation": "special/ultra",
```

Comment:
I probably would have done special_or_ultra, but this is fine i guess.

Reply:
in fact, yours probably works better if you want to just use the string as is.

Reply:
yeah, mostly went with this for the string appearance